@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -873,9 +884,22 @@ export class TiltViewClipPlanesEvaluator extends TopViewClipPlanesEvaluator {
         }
         viewRanges.far = farPlane;
 
+        // Compute the focus point (target) distance for current camera and projection setup,
+        // in a same way the MapView component does.
+        // TODO: Target distance does not need to be cached when we provide full support for
+        // target point within all camera tilt angles.
+        const target = MapViewUtils.getWorldTargetFromCamera(camera, projection);
+        if (target !== null) {
+            this.m_targetDistance = camera.position.distanceTo(target);
+        }
+        // Fallback to camera altitude if target not found and not yet set.
+        // NOTE: May be inaccurate when launching map with camera looking to void.
+        else if (this.m_targetDistance < cameraAltitude) {
+            this.m_targetDistance = cameraAltitude;
+        }
         // Apply the constraints.
         const farMin = cameraAltitude - this.minElevation;
-        const farMax = mapView.targetDistance * this.farMaxRatio;
+        const farMax = this.m_targetDistance * this.farMaxRatio;
```

Comment:
Can you please remove `targetDistance` from the `MapView` API before people start using it

Reply:
I see, so leave it for now. I will do it when making ViewState a member of MapView instead of a proxy like it is right now.